src/server.js

```diff
@@ -0,0 +1,35 @@
+import 'dotenv/config'
+import { WebSocketServer } from 'ws'
+import { v4 as uuid } from 'uuid'
+import ZmqConnection from './zmq.js'
+import Client from './client.js'
+import Dashcore from '@dashevo/dashcore-lib'
+
+export default class Server {
+  #clients = []
+
+  constructor () {
+    this.zmq = new ZmqConnection()
+    this.zmq.on('rawtx', (data) => {
+      this.handleRawTransactions(data)
```

Comment:
`handleRawTx`

Reply:
or `handleRawTransaction`

Reply:
also, I believe that you can use function reference instead of arrow function.

`this.zmq.on('rawtx', this.handleRawTx)`

Reply:
yes  it’s working, but in that case  we need to bind the class , so we can use the arrow function syntax and avoid additionally setting this.handleRaw = this.handleRaw.bind(this)